common/src/main/java/com/microsoft/identity/common/internal/request/SdkType.java

```diff
@@ -27,5 +27,19 @@
  */
 public enum SdkType {
     ADAL,
-    MSAL
+    MSAL,
+    MSALCPP;
+
+    private final String PRODUCT_NAME_MSAL = "MSAL.Android";
+
+    private final String PRODUCT_NAME_MSAL_CPP = "MSAL.xplat.Android";
```

Comment:
Move this to Authentication Constants

Reply:
[https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/1034/commits/45fba567fbf3c42c2a50bb4229b00b969db134b4](url)